NEL.MESH/Services/Foundations/Mesh/MeshService.cs

```diff
@@ -37,19 +38,56 @@ public MeshService(IMeshBroker meshBroker)
             {
                 ValidateMeshMessageOnSendMessage(message, authorizationToken);
 
-                HttpResponseMessage responseMessage = await this.meshBroker.SendMessageAsync(
-                    mailboxTo: GetKeyStringValue("Mex-To", message.Headers),
-                    workflowId: GetKeyStringValue("Mex-WorkflowID", message.Headers),
-                    localId: GetKeyStringValue("Mex-LocalID", message.Headers),
-                    subject: GetKeyStringValue("Mex-Subject", message.Headers),
-                    fileName: GetKeyStringValue("Mex-FileName", message.Headers),
-                    contentChecksum: GetKeyStringValue("Mex-Content-Checksum", message.Headers),
-                    contentEncrypted: GetKeyStringValue("Mex-Content-Encrypted", message.Headers),
-                    encoding: GetKeyStringValue("Mex-Encoding", message.Headers),
-                    chunkRange: GetKeyStringValue("Mex-Chunk-Range", message.Headers),
-                    contentType: GetKeyStringValue("Content-Type", message.Headers),
-                    authorizationToken,
-                    stringContent: message.StringContent);
+                string chunkRange = GetKeyStringValue("Mex-Chunk-Range", message.Headers)
```

Comment:
Mex-Chunk-Range is an optional header.    Do a check here to see if the value is string.empty.   If it is empty, then set the value to 1 as we can then assume that it is a single part message.